listenbrainz/webserver/views/settings.py

```diff
@@ -242,6 +233,36 @@ def refresh_service_token(service_name: str):
     return jsonify({"access_token": user["access_token"]})
 
 
+@settings_bp.route('/music-services/<service_name>/connect/', methods=['POST'])
```

Comment:
The last.fm connections differ from other services because of the absence of the oauth workflow or options on how the connection should work. We can look into separating the disconnect endpoint into two but will need to figure out and ensure that you can switch the permissions granted to a service in one click on LB.